src/questions.js

```diff
@@ -0,0 +1,156 @@
+const starterQuestions = [
+  {
+    type: "input",
+    name: "title",
+    message: "What is the title of your project?",
+  },
+  {
+    type: "input",
+    name: "description",
+    message: "Provide a description for your Project",
+  },
+  {
```

Comment:
Although this is a good use of an inquirer question type, it adds unnecessary complexity to the app and question structures. It would have been better to ask 3 separate confirm questions here, and then using the inquirer `when` key to only run the next question (so the question asking the user to add the first step) WHEN the confirm question came back as true:

```
  {
    type: "confirm",
    message: "Does your project require any installation?",
    name: "installation",
  },

  {
    type: "input",
    message: "Please add the necessary installation information.",
    name: "installationInformation",
    when: (answers) => {
      return answers.installation;
    },
  },
```